src/site/_data/tagsData.json

```diff
@@ -131,6 +131,10 @@
   "media-queries": {
     "title": "Media Queries"
   },
+  "memory": {
+    "title": "Memory",
+    "description": "News and guidance related to memory management on the web."
```

Comment:
```suggestion
    "description": "Memory management on the web."
```

Reply:
If we wrote "News and guidance related to" on everything It would be something readers would have to wade through instead of being helpful.